### PR TITLE
add ndd

### PR DESCRIPTION
**Pull request by simomarsili**
Hi, this is a python/fortran repo.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by simomarsili** [comment]
ready for review

File: recipes/ndd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 884c2dedefd48b4cb85be379dce0b70d6e59dc68638ea2d87564efdcb5bee5cc
+
+build:
+  number: 3
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - numpy >=1.13
```
Comment (dopplershift) [inline recipes/ndd/meta.yaml:19]: You should only need numpy in `host`

File: recipes/ndd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simomarsili/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 884c2dedefd48b4cb85be379dce0b70d6e59dc68638ea2d87564efdcb5bee5cc
+
+build:
+  number: 3
```
Comment (dopplershift) [inline recipes/ndd/meta.yaml:13]: Build number should be 0 to start.

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/ndd/meta.yaml (hunk 1 above)
Comment (simomarsili) [reply recipes/ndd/meta.yaml]: Thanks!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**